backend/infrahub/core/schema/schema_branch.py

```diff
@@ -327,13 +341,22 @@ def get_profile(self, name: str, duplicate: bool = True) -> ProfileSchema:
             raise ValueError(f"{name!r} is not of type ProfileSchema")
         return item
 
+    def get_template(self, name: str, duplicate: bool = True) -> TemplateSchema:
```

Comment:
Should it be `kind` instead of `name`? Same for `SchemaBranch.get`?

Reply:
I used the same parameter name as all the methods that perform the same kind of operations. I agree that `name` is not the best name for it, I used it for consistency reasons. I guess we could rename that parameter for all the methods in another PR.